sympy/benchmarks/mechanics/kane_1.py

```diff
@@ -0,0 +1,20 @@
+# This is taken from the example in KanesMethod docstring
+from sympy import symbols
+import sympy.physics.mechanics as me
+
+q, u = me.dynamicsymbols('q u')
+qd, ud = me.dynamicsymbols('q u', 1)
+m, c, k = symbols('m c k')
+N = me.ReferenceFrame('N')
+P = me.Point('P')
+P.set_vel(N, u * N.x)
+
+kd = [qd - u]
+FL = [(P, (-k * q - c * u) * N.x)]
+pa = me.Particle('pa', P, m)
+BL = [pa]
+
+
+def kmethod():
+    KM = me.KanesMethod(N, q_ind=[q], u_ind=[u], kd_eqs=kd)
+    (fr, frstar) = KM.kanes_equations(BL, FL)
```

Comment:
This is a nice simple use case to benchmark. Couple comments:

1. It may be worth moving the general symbolic models from `pydy.models` into SymPy so we can use them for benchmarking and testing purposes.
2. We need an example that exercises all of the KanesMethod features (or several models that hit all the options) at some point. The best example we have is the Whipple bicycle model, but it is currently failing to linearize. The nonlinear derivation in test_kane3.py can be used though. See the bicycle/whipple PR in pydy to for an alternative example of this. The rolling disc examples do this pretty well too.